opal/browser/dom/node.rb

```diff
@@ -235,17 +230,19 @@ def cdata?
   # @!attribute [r] child
   # @return [Node?] the first child of the node
   def child
```

Comment:
Currently in DOM spec for `ParentNode` there are only 4 attributes defined and other attributes like `firstChild`, `lastChild` are defined for `Node` and not for `ParentNode` so I'm not really sure, but it can be moved.


I think implementation should follow as close to DOM spec as possible (while also giving convenience methods) and there's [ParentNode Interface](https://dom.spec.whatwg.org/#interface-parentnode) so this follows it.